app/controllers/ansible_roles_controller.rb

```diff
@@ -22,18 +22,19 @@ def destroy
 
   def import
     changed = @importer.import!
-    if changed.values.all?(&:empty?)
+    @rows = prepare_ansible_import_rows(changed, @variables_importer)
+    if @rows.empty?
       success no_changed_roles_message
```

Comment:
this message could be tailored into "No added or removed roles nor variables" I'm ok to do so in another PR though :)

Reply:
done